src/app/components_ngrx/work-item-detail/work-item-detail.component.html

```diff
@@ -271,7 +271,7 @@
                 [rawText]="workItem.description"
                 [renderedText]="workItem.descriptionRendered"
                 (onSaveClick)="descUpdate($event)"
-                (showPreview)="showPreview($event)">
+                (showPreview)="showPreview($event)" #markdown>
```

Comment:
The name has to be more specific. With dynamic fields coming in, there can be multiple markdown fields.